src/widgets/modals/CashTransactionModal.js

```diff
@@ -38,7 +43,7 @@ export const CashTransactionModal = ({ onConfirm }) => {
   const [isReasonModalOpen, setIsReasonModalOpen] = useState(false);
   const [isDescriptionModalOpen, setIsDescriptionModalOpen] = useState(false);
 
-  const names = useMemo(() => UIDatabase.objects('Name'));
+  const names = useMemo(() => UIDatabase.objects('Name').filtered('isVisible == true'));
```

Comment:
sorry missed this before - Should probably remove the mobile stores name, and also `InventoryAdjustment`, `build` and `repack` names. Could potentially use all `isCustomer`, `isSupplier` and `isPatient` names, I'm pretty sure. Possibly worth a `UIDatabase` case.

Reply:
5498cbe109c2a4a5b0d05cc5392abb00a9828b2f.

Reply:
`.filtered("isSupplier == $0 || isCustomer == $0 || isPatient == $0",true)` is that nicer? I'm not sure but it certainly isn't worth even the time it took me to type it!

Reply:
A bit nicer, but equally arguable that should only parametrise queries when parameter is variable (constant here). Will leave for now but don't feel strongly either way.